scripts/checker.py

```diff
@@ -75,27 +82,44 @@ def do_check(row):
     context["cs_path_info"] = row["path"]
     context["cs_username"] = row["username"]
     context["cs_user_info"] = {"username": row["username"]}
+    context['is_running_checker'] = True			# may be used by preload.py
```

Comment:
I'm not sure I understand this piece.  How is `is_running_checker` expected to be used?  I don't see it referenced anywhere else in this PR.

Reply:
no longer needed - removing